### ISSUE TITLE
Beets catalog

### ISSUE THREAD

**Issue by kuzi-moto** [ampache/ampache-docker#75]
Should we include Beets so that the beets catalog type works? Requires Beets to be installed to work properly.

**Comment by tgdnt** [comment]
@lachlan-00 thanks for your work on this. I saw [your commit](https://github.com/ampache/ampache-docker/commit/8b34614d93d16136cb72e087aa57801a43bb0d28), but I don't see beets added there, only logrotate and when I tested, I could not use a beets catalog. I edited the Dockerfile and tested it, and that is the only thing missing to get beets catalogs working with the docker deployment. Do you want me to create a PR to close this ticket with?

**Comment by lachlan-00** [comment]
might help to actually ADD the package...

**Comment by lachlan-00** [comment]
i'm building new images for 6.3.0 with beets added to the dockerfile

**Comment by edgar-vincent** [comment]
Should this issue be closed? It seems beets has been added to the image.

**Comment by tgdnt** [comment]
Yes, it should be closed. This was the commit: c916e417f59a6fadd3b35dd205f252761c6870c0
